sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -13,8 +14,14 @@ export default abstract class HttpBaseClient {
       throw new Error("User is not authenticated!");
     }
   }
+  static renewTokenIfExpired() {
+    if (AuthService.shared.keycloak?.isTokenExpired(MINIMUM_TOKEN_VALIDITY)) {

```

Comment:
What happens if the user stays active in the application without submitting any API call for 30min and the refresh token expires? For instance, filling the Student Application but not saving or submitting it?
The refresh token will expire, right?

Reply:
I too have the same question.

Reply:
updated as discussed